.circleci/config.yml

```diff
@@ -291,13 +291,13 @@ jobs:
           command: |
             python -m venv env
             . env/bin/activate
-            pip install --upgrade << parameters.dependency-versions >>
+            pip install --upgrade setuptools pip wheel
```

Comment:
Ideally we would test without setuptools, as it's not available by default in 3.12. (And it's specified as a build dependency in pyproject.toml already).

Reply:
Let's [see how it goes](https://github.com/dwavesystems/minorminer/pull/239/commits/aca884457b1bd1888b0d9ba88ceafd8159431050)